pkg/controllers/utils/expectations/active_expectation_test.go

```diff
@@ -150,4 +178,66 @@ func TestActiveExpectations(t *testing.T) {
 	expectation, err = exp.GetExpectation(pod.Namespace, pod.Name)
 	g.Expect(err).Should(gomega.BeNil())
 	g.Expect(expectation).Should(gomega.BeNil())
+
+	g.Expect(exp.ExpectCreate(pod, Pod, podA.Name)).Should(gomega.BeNil())
+	g.Expect(exp.IsSatisfied(pod)).Should(gomega.BeFalse())
+	g.Expect(exp.DeleteItem(pod, Pod, podA.Name)).Should(gomega.BeNil())
+	g.Expect(exp.IsSatisfied(pod)).Should(gomega.BeTrue())
+	// delete no existing item should not get error
```

Comment:
should not get error when delete not existing item
or
deleting not existing item should not get error